rank=11 L00: Traceback (most recent call last):
rank=11 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=11 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=11 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=11 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=11 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=11 L06:     f(args)
rank=11 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=11 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=11 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=11 L10:     return _run_module_code(code, init_globals, run_name,
rank=11 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=11 L12:     _run_code(code, mod_globals, init_globals,
rank=11 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=11 L14:     exec(code, run_globals)
rank=11 L15:   File "get_ph.py", line 16, in <module>
rank=11 L16:     ph.read(symmetrize=10, acoustic=True)
rank=11 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=11 L18:     fplus_av = pickleload(fd)
rank=11 L19: EOFError: Ran out of input
rank=06 L00: Traceback (most recent call last):
rank=06 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=06 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=06 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=06 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=06 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=06 L06:     f(args)
rank=06 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=06 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=06 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=06 L10:     return _run_module_code(code, init_globals, run_name,
rank=06 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=06 L12:     _run_code(code, mod_globals, init_globals,
rank=06 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=06 L14:     exec(code, run_globals)
rank=06 L15:   File "get_ph.py", line 16, in <module>
rank=06 L16:     ph.read(symmetrize=10, acoustic=True)
rank=06 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=06 L18:     fplus_av = pickleload(fd)
rank=06 L19: EOFError: Ran out of input
GPAW CLEANUP (node 6): <class 'EOFError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 11): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=18 L00: Traceback (most recent call last):
rank=18 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=18 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=18 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=18 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=18 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=18 L06:     f(args)
rank=18 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=18 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=18 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=18 L10:     return _run_module_code(code, init_globals, run_name,
rank=18 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=18 L12:     _run_code(code, mod_globals, init_globals,
rank=18 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=18 L14:     exec(code, run_globals)
rank=18 L15:   File "get_ph.py", line 16, in <module>
rank=18 L16:     ph.read(symmetrize=10, acoustic=True)
rank=18 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=18 L18:     fplus_av = pickleload(fd)
rank=18 L19: EOFError: Ran out of input
GPAW CLEANUP (node 18): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=13 L00: Traceback (most recent call last):
rank=13 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=13 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=13 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=13 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=13 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=13 L06:     f(args)
rank=13 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=13 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=13 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=13 L10:     return _run_module_code(code, init_globals, run_name,
rank=13 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=13 L12:     _run_code(code, mod_globals, init_globals,
rank=13 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=13 L14:     exec(code, run_globals)
rank=13 L15:   File "get_ph.py", line 16, in <module>
rank=13 L16:     ph.read(symmetrize=10, acoustic=True)
rank=13 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=13 L18:     fplus_av = pickleload(fd)
rank=13 L19: EOFError: Ran out of input
rank=07 L00: Traceback (most recent call last):
rank=07 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=07 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=07 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=07 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=07 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=07 L06:     f(args)
rank=07 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=07 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=07 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=07 L10:     return _run_module_code(code, init_globals, run_name,
rank=07 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=07 L12:     _run_code(code, mod_globals, init_globals,
rank=07 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=07 L14:     exec(code, run_globals)
rank=07 L15:   File "get_ph.py", line 16, in <module>
rank=07 L16:     ph.read(symmetrize=10, acoustic=True)
rank=07 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=07 L18:     fplus_av = pickleload(fd)
rank=07 L19: EOFError: Ran out of input
rank=03 L00: Traceback (most recent call last):
rank=03 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=03 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=03 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=03 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=03 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=03 L06:     f(args)
rank=03 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=03 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=03 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=03 L10:     return _run_module_code(code, init_globals, run_name,
rank=03 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=03 L12:     _run_code(code, mod_globals, init_globals,
rank=03 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=03 L14:     exec(code, run_globals)
rank=03 L15:   File "get_ph.py", line 16, in <module>
rank=03 L16:     ph.read(symmetrize=10, acoustic=True)
rank=03 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=03 L18:     fplus_av = pickleload(fd)
rank=03 L19: EOFError: Ran out of input
rank=08 L00: Traceback (most recent call last):
rank=08 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=08 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=08 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=08 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=08 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=08 L06:     f(args)
rank=08 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=08 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=08 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=08 L10:     return _run_module_code(code, init_globals, run_name,
rank=08 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=08 L12:     _run_code(code, mod_globals, init_globals,
rank=08 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=08 L14:     exec(code, run_globals)
rank=08 L15:   File "get_ph.py", line 16, in <module>
rank=08 L16:     ph.read(symmetrize=10, acoustic=True)
rank=08 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=08 L18:     fplus_av = pickleload(fd)
rank=08 L19: EOFError: Ran out of input
rank=14 L00: Traceback (most recent call last):
rank=14 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=14 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=14 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=14 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=14 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=14 L06:     f(args)
rank=14 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=14 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=14 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=14 L10:     return _run_module_code(code, init_globals, run_name,
rank=14 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=14 L12:     _run_code(code, mod_globals, init_globals,
rank=14 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=14 L14:     exec(code, run_globals)
rank=14 L15:   File "get_ph.py", line 16, in <module>
rank=14 L16:     ph.read(symmetrize=10, acoustic=True)
rank=14 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=14 L18:     fplus_av = pickleload(fd)
rank=14 L19: EOFError: Ran out of input
GPAW CLEANUP (node 14): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=04 L00: Traceback (most recent call last):
rank=04 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=04 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=04 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=04 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=04 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=04 L06:     f(args)
rank=04 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=04 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=04 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=04 L10:     return _run_module_code(code, init_globals, run_name,
rank=04 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=04 L12:     _run_code(code, mod_globals, init_globals,
rank=04 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=04 L14:     exec(code, run_globals)
rank=04 L15:   File "get_ph.py", line 16, in <module>
rank=04 L16:     ph.read(symmetrize=10, acoustic=True)
rank=04 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=04 L18:     fplus_av = pickleload(fd)
rank=04 L19: EOFError: Ran out of input
GPAW CLEANUP (node 8): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=02 L00: Traceback (most recent call last):
rank=02 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=02 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=02 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=02 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=02 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=02 L06:     f(args)
rank=02 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=02 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=02 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=02 L10:     return _run_module_code(code, init_globals, run_name,
rank=02 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=02 L12:     _run_code(code, mod_globals, init_globals,
rank=02 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=02 L14:     exec(code, run_globals)
rank=02 L15:   File "get_ph.py", line 16, in <module>
rank=02 L16:     ph.read(symmetrize=10, acoustic=True)
rank=02 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=02 L18:     fplus_av = pickleload(fd)
rank=02 L19: EOFError: Ran out of input
rank=09 L00: Traceback (most recent call last):
rank=09 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=09 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=09 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=09 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=09 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=09 L06:     f(args)
rank=09 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=09 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=09 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=09 L10:     return _run_module_code(code, init_globals, run_name,
rank=09 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=09 L12:     _run_code(code, mod_globals, init_globals,
rank=09 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=09 L14:     exec(code, run_globals)
rank=09 L15:   File "get_ph.py", line 16, in <module>
rank=09 L16:     ph.read(symmetrize=10, acoustic=True)
rank=09 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=09 L18:     fplus_av = pickleload(fd)
rank=09 L19: EOFError: Ran out of input
rank=16 L00: Traceback (most recent call last):
rank=16 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=16 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=16 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=16 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=16 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=16 L06:     f(args)
rank=16 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=16 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=16 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=16 L10:     return _run_module_code(code, init_globals, run_name,
rank=16 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=16 L12:     _run_code(code, mod_globals, init_globals,
rank=16 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=16 L14:     exec(code, run_globals)
rank=16 L15:   File "get_ph.py", line 16, in <module>
rank=16 L16:     ph.read(symmetrize=10, acoustic=True)
rank=16 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=16 L18:     fplus_av = pickleload(fd)
rank=16 L19: EOFError: Ran out of input
GPAW CLEANUP (node 4): <class 'EOFError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 13): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=05 L00: Traceback (most recent call last):
rank=05 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=05 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=05 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=05 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=05 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=05 L06:     f(args)
rank=05 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=05 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=05 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=05 L10:     return _run_module_code(code, init_globals, run_name,
rank=05 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=05 L12:     _run_code(code, mod_globals, init_globals,
rank=05 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=05 L14:     exec(code, run_globals)
rank=05 L15:   File "get_ph.py", line 16, in <module>
rank=05 L16:     ph.read(symmetrize=10, acoustic=True)
rank=05 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=05 L18:     fplus_av = pickleload(fd)
rank=05 L19: EOFError: Ran out of input
GPAW CLEANUP (node 5): <class 'EOFError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 9): <class 'EOFError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 2): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=10 L00: Traceback (most recent call last):
rank=10 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=10 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=10 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=10 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=10 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=10 L06:     f(args)
rank=10 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=10 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=10 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=10 L10:     return _run_module_code(code, init_globals, run_name,
rank=10 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=10 L12:     _run_code(code, mod_globals, init_globals,
rank=10 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=10 L14:     exec(code, run_globals)
rank=10 L15:   File "get_ph.py", line 16, in <module>
rank=10 L16:     ph.read(symmetrize=10, acoustic=True)
rank=10 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=10 L18:     fplus_av = pickleload(fd)
rank=10 L19: EOFError: Ran out of input
GPAW CLEANUP (node 16): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=12 L00: Traceback (most recent call last):
rank=12 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=12 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=12 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=12 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=12 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=12 L06:     f(args)
rank=12 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=12 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=12 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=12 L10:     return _run_module_code(code, init_globals, run_name,
rank=12 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=12 L12:     _run_code(code, mod_globals, init_globals,
rank=12 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=12 L14:     exec(code, run_globals)
rank=12 L15:   File "get_ph.py", line 16, in <module>
rank=12 L16:     ph.read(symmetrize=10, acoustic=True)
rank=12 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=12 L18:     fplus_av = pickleload(fd)
rank=12 L19: EOFError: Ran out of input
rank=15 L00: Traceback (most recent call last):
rank=15 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=15 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=15 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=15 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=15 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=15 L06:     f(args)
rank=15 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=15 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=15 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=15 L10:     return _run_module_code(code, init_globals, run_name,
rank=15 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=15 L12:     _run_code(code, mod_globals, init_globals,
rank=15 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=15 L14:     exec(code, run_globals)
rank=15 L15:   File "get_ph.py", line 16, in <module>
rank=15 L16:     ph.read(symmetrize=10, acoustic=True)
rank=15 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=15 L18:     fplus_av = pickleload(fd)
rank=15 L19: EOFError: Ran out of input
GPAW CLEANUP (node 7): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=17 L00: Traceback (most recent call last):
rank=17 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=17 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=17 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=17 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=17 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=17 L06:     f(args)
rank=17 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=17 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=17 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=17 L10:     return _run_module_code(code, init_globals, run_name,
rank=17 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=17 L12:     _run_code(code, mod_globals, init_globals,
rank=17 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=17 L14:     exec(code, run_globals)
rank=17 L15:   File "get_ph.py", line 16, in <module>
rank=17 L16:     ph.read(symmetrize=10, acoustic=True)
rank=17 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=17 L18:     fplus_av = pickleload(fd)
rank=17 L19: EOFError: Ran out of input
GPAW CLEANUP (node 10): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=19 L00: Traceback (most recent call last):
rank=19 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=19 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=19 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=19 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=19 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=19 L06:     f(args)
rank=19 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=19 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=19 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=19 L10:     return _run_module_code(code, init_globals, run_name,
rank=19 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=19 L12:     _run_code(code, mod_globals, init_globals,
rank=19 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=19 L14:     exec(code, run_globals)
rank=19 L15:   File "get_ph.py", line 16, in <module>
rank=19 L16:     ph.read(symmetrize=10, acoustic=True)
rank=19 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=19 L18:     fplus_av = pickleload(fd)
rank=19 L19: EOFError: Ran out of input
GPAW CLEANUP (node 12): <class 'EOFError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 15): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=01 L00: Traceback (most recent call last):
rank=01 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=01 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=01 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=01 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=01 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=01 L06:     f(args)
rank=01 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=01 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=01 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=01 L10:     return _run_module_code(code, init_globals, run_name,
rank=01 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=01 L12:     _run_code(code, mod_globals, init_globals,
rank=01 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=01 L14:     exec(code, run_globals)
rank=01 L15:   File "get_ph.py", line 16, in <module>
rank=01 L16:     ph.read(symmetrize=10, acoustic=True)
rank=01 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=01 L18:     fplus_av = pickleload(fd)
rank=01 L19: EOFError: Ran out of input
GPAW CLEANUP (node 17): <class 'EOFError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 3): <class 'EOFError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 1): <class 'EOFError'> occurred.  Calling MPI_Abort!
GPAW CLEANUP (node 19): <class 'EOFError'> occurred.  Calling MPI_Abort!
rank=00 L00: Traceback (most recent call last):
rank=00 L01:   File "/home/niflheim2/cmr/WIP/stacking/venv/bin/gpaw", line 33, in <module>
rank=00 L02:     sys.exit(load_entry_point('gpaw', 'console_scripts', 'gpaw')())
rank=00 L03:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/main.py", line 80, in main
rank=00 L04:     ase_main('gpaw', 'GPAW command-line tool', __version__,
rank=00 L05:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/cli/main.py", line 103, in main
rank=00 L06:     f(args)
rank=00 L07:   File "/home/niflheim2/cmr/WIP/stacking/venv/gpaw/gpaw/cli/python.py", line 34, in run
rank=00 L08:     runpy.run_path(args.arguments[0], run_name='__main__')
rank=00 L09:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 265, in run_path
rank=00 L10:     return _run_module_code(code, init_globals, run_name,
rank=00 L11:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 97, in _run_module_code
rank=00 L12:     _run_code(code, mod_globals, init_globals,
rank=00 L13:   File "/home/modules/software/Python/3.8.6-GCCcore-10.2.0/lib/python3.8/runpy.py", line 87, in _run_code
rank=00 L14:     exec(code, run_globals)
rank=00 L15:   File "get_ph.py", line 16, in <module>
rank=00 L16:     ph.read(symmetrize=10, acoustic=True)
rank=00 L17:   File "/home/niflheim2/cmr/WIP/stacking/venv/ase/ase/phonons.py", line 439, in read
rank=00 L18:     fplus_av = pickleload(fd)
rank=00 L19: EOFError: Ran out of input
GPAW CLEANUP (node 0): <class 'EOFError'> occurred.  Calling MPI_Abort!
Abort(42) on node 9 (rank 9 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 9
Abort(42) on node 14 (rank 14 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 14
Abort(42) on node 10 (rank 10 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 10
Abort(42) on node 11 (rank 11 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 11
Abort(42) on node 16 (rank 16 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 16
Abort(42) on node 13 (rank 13 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 13
Abort(42) on node 7 (rank 7 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 7
Abort(42) on node 17 (rank 17 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 17
Abort(42) on node 18 (rank 18 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 18
Abort(42) on node 8 (rank 8 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 8
Abort(42) on node 3 (rank 3 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 3
Abort(42) on node 19 (rank 19 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 19
Abort(42) on node 5 (rank 5 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 5
Abort(42) on node 12 (rank 12 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 12
Abort(42) on node 15 (rank 15 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 15
Abort(42) on node 6 (rank 6 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 6
Abort(42) on node 0 (rank 0 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 0
Abort(42) on node 4 (rank 4 in comm 0): application called MPI_Abort(MPI_COMM_WORLD, 42) - process 4
